.pre-commit-config.yaml

```
@@ -109,6 +109,8 @@ repos:
     hooks:
       - id: mypy
         additional_dependencies:
+          - types-requests
+          - types-PyYAML
```

Prefer alphabetical order:
```suggestion
          - types-PyYAML
          - types-requests
```